### ISSUE TITLE
Train my own dataset?

### ISSUE THREAD

**Issue by ZengyuanYu** [JihongJu/keras-fcn#36]
Now I have some pictures, but how i create label(labelme?) ? And then  how I train this datasets?

**Comment by ShreyasAiyar** [comment]
Yes, you can label images using labelme (python). Your original images form your X_train and your labelled images are y_train. Subsequently you can apply data augmentation as well if you lack sufficient training examples.
